mirage/config.js

```diff
@@ -408,6 +419,12 @@ export default function() {
 
     let [sluggedRoute] = schema.sluggedRoutes.where({ 'slug': sluggedRouteSlug }).models;
 
+    if (!sluggedRoute) {
+      // If there's no slugged route, we likely added a new route and didn't
+      // `routes` array above.
+      console.error(`Did you add a new route under /${sluggedRouteSlug} (/${projectSlug}) without adding it to mirage/config.js?`);
+    }
```

Comment:
I like this 👍

Reply:
I do too after losing 90 mins yesterday figuring out what was wrong. :facepalm: